gulpfile.js

```diff
@@ -37,7 +42,37 @@ function lessThemes () {
     .pipe(gulp.dest('source/common/assets/css/'))
 }
 
+/**
+ * Precompile handlebar templates
+ */
+function handlebarTemplates () {
+  return gulp.src(resourcesTemplatesDir + '**/*.handlebars')
```

Comment:
I would leave out resources and instead use only `tplDirectory` and `lessDirectory` (it's more consistent with the rest of the app's naming conventions, and "resources" is a little bit doubled, as gulp is basically the resources-to-assets-compiler file — this should also be something for us to keep in mind: Everything that needs to be compiled needs to be put in resources, and compiled to one of the assets directory, may help clarify things in future decisions!)

Reply:
Ok, changed.